jx_sqlite/query_table.py

```diff
@@ -276,7 +281,10 @@ def query(self, query):
                 row = [None for _ in column_names]
                 for s in index_to_columns.values():
                     if s.push_child == ".":
-                        row[s.push_column] = s.pull(d)
+                        if not row[s.push_column]:                        
+                            row[s.push_column] = s.pull(d)
+                        else:
+                            row[s.push_column] += s.pull(d)
```

Comment:
maybe I do not understand what you are doing here (and elsewhere); I hope it is not the side effect of counting!

Be sure ES  does the work of aggregation.